content/pt/docs/reference/access-authn-authz/bootstrap-tokens.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Autenticando com Tokens de Inicialização
+content_type: concept
+weight: 20
+---
+ 
+<!-- overview -->
+ 
+{{< feature-state for_k8s_version="v1.18" state="stable" >}}
+ 
+Os tokens de inicialização são um _bearer token_ simples que devem ser utilizados
+ao criar novos clusters ou para quando novos nós são registrados a clusters existentes. Eles foram construídos
+para suportar a ferramenta [kubeadm](/docs/reference/setup-tools/kubeadm/), mas podem ser utilizados em outros contextos para usuários que desejam inicializar clusters sem utilizar o `kubeadm`.
+Foram também construídos para funcionar, via políticas RBAC, com o sistema de [Inicialização do Kubelet via TLS](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/).
+ 
+<!-- body -->
+## Visão geral dos tokens de inicialização
+ 
+Os tokens de inicialização são definidos com um tipo especifico de segredos (`bootstrap.kubernetes.io/token`) que existem no namespace `kube-system`. Estes segredos são então lidos pelo autenticador de inicialização do servidor de API.
+Tokens expirados são removidos pelo controlador _TokenCleaner_ no Gerenciador de Controle.
+Os tokens também são utilizados para criar uma assinatura para um ConfigMap específico usado no processo de descoberta através de um controlador denominado `BootstrapSigner`.
+ 
+## Formato do Token
+ 
+Tokens de inicialização tem o formato `abcdef.0123456789abcdef`. Mais formalmente, eles devem corresponder a expressão regular `[a-z0-9]{6}\.[a-z0-9]{16}`.
+ 
+A primeira parte do token é um identificador ("Token ID") e é considerado informação pública.
+Ele é utilizado para se referir a um token sem vazar a parte secreta usada para autenticação.
+A segunda parte é o segredo do token e somente deve ser compartilhado com partes confiáveis.
+ 
+## Habilitando autenticação com tokens de inicialização
+ 
+O autenticador de tokens de inicialização pode ser habilitado utilizando a seguinte opção no servidor de API:
+ 
+```
+--enable-bootstrap-token-auth
+```
+ 
+Quando habilitado, tokens de inicialização podem ser utilizado como credenciais _bearer token_
+para autenticar requisições no servidor de API.
+ 
+```http
+Authorization: Bearer 07401b.f395accd246ae52d
+```
+ 
+Tokens são autenticados como o usuário `system:bootstrap:<token id>` e são membros
+do grupo `system:bootstrappers`. Grupos adicionais podem ser
+especificados dentro do segredo do token.
+ 
+Tokens expirados podem ser removidos automaticamente ao habilitar o controlador `tokencleaner`
+do gerenciador de controle.
```

Comment:
here, please make a reference to kube-controller-manager (I don't know if there's a conscensus about translating the words, so we can at least say "gerenciador de controle - _kube-controller-manager_"